multiqc/modules/fastqc/fastqc.py

```diff
@@ -556,7 +557,52 @@ def seq_dup_levels_plot (self):
                         plots.linegraph.plot(data, pconfig)
         })
         
-        
+    
+    def overrepresented_sequences (self):
+        """Sum the percentages of overrepresented sequences and display them in a bar plot"""
+
+        data = dict()
+        for s_name in self.fastqc_data:
+            data[s_name]=dict()
```

Comment:
If you move this into the `try` block (after calculating `total_pcnt`), the sample key won't be created for samples that don't have this data. Then you won't need the `try`/`except` block later on in the code (L586)